### PR TITLE
Fixes macro comment issues

### PR DESCRIPTION
**Pull request by anselmbradford**
Fixes issues in code comments in macros.

## Changes

- Fixed typo,  incorrectly identified methods.

## Review

- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim 

## Notes

- Sprint 30 PR cleanup.

### CODE REVIEW

File: _includes/macros/activity-snippets.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 {# ==========================================================================
 
-   _activity_snippet()
```
Comment (sebworks) [inline _includes/macros/activity-snippets.html]: Do we need `PRIVATE`?

Comment (anselmbradford) [inline _includes/macros/activity-snippets.html]: Not necessarily. But if not here it should probably be mentioned as private in the code comment description.

Comment (sebworks) [inline _includes/macros/activity-snippets.html]: I don't think it's necessary. In javascript we rely upon the underscore to denote something as private. It should be the same in Jinja. `Macros and variables starting with one or more underscores are private and cannot be imported.`

Comment (anselmbradford) [inline _includes/macros/activity-snippets.html]: :+1: SG, fixed in https://github.com/cfpb/cfgov-refresh/commit/4cf0a2537d336fde83866b5074a7ff716acfcfeb

**Comment by sebworks** [comment]
:+1:

**Pull request merged**